charts/prometheus-artifactory-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.14.0"
 description: A Helm chart for the Prometheus Artifactory Exporter
 name: prometheus-artifactory-exporter
-version: 0.6.4
+version: 0.6.5
```

Bumping minor as there is a new feature and new app version in this change
```suggestion
version: 0.7.0
```